src/main/java/fshop/Main.java

```diff
@@ -0,0 +1,28 @@
+package fshop;
+
+import fshop.db.FoodDao;
+import fshop.db.FoodDaoImpl;
+import fshop.service.workfile.ReadCsv;
+import fshop.service.workfile.ReadCsvImpl;
+import fshop.service.workfile.WriteCsv;
+import fshop.service.workfile.WriteCsvImpl;
+
+public class Main {
+    public static void main(String[] args) {
+        ReadCsv readCsvFirst = new ReadCsvImpl("file.csv");
+        FoodDao foodDaoFirst = new FoodDaoImpl(readCsvFirst);
+        foodDaoFirst.addAll();
+        foodDaoFirst.updateAll();
```

Comment:
Let's exchange the information between services but not put service into another service:
```suggestion
        ReadCsv readCsvFirst = new ReadCsvImpl()
        List<String> fileLines = readCsvFirst.readData("file.csv");
        FoodDao foodDaoFirst = new FoodDaoImpl();
        foodDaoFirst.addAll(fileLines);
        foodDaoFirst.updateAll(fileLines);
```
Something like this, depends on your particular logic;

Reply:
done, cool!